plugins/Microwave/Microwave.h

```diff
@@ -0,0 +1,1125 @@
+	FloatModel * m_vol[8];
+	FloatModel * m_pan[8];
+	FloatModel * m_detune[8];
+	FloatModel * m_phase[8];
+	FloatModel * m_phaseRand[8];
+	BoolModel * m_enabled[8];
+	BoolModel * m_muted[8];
+	FloatModel * m_sampLen[8];
+	FloatModel * m_morphMax[8];
+	FloatModel * m_unisonVoices[8];
+	FloatModel * m_unisonDetune[8];
+	FloatModel * m_unisonMorph[8];
+	FloatModel * m_unisonModify[8];
+	BoolModel * m_keytracking[8];
+	FloatModel * m_tempo[8];
+	BoolModel * m_interpolate[8];
+
+	BoolModel * m_subEnabled[64];
+	BoolModel * m_subMuted[64];
+	BoolModel * m_subKeytrack[64];
+	BoolModel * m_subNoise[64];
+	FloatModel * m_subVol[64];
+	FloatModel * m_subPanning[64];
+	FloatModel * m_subDetune[64];
+	FloatModel * m_subPhase[64];
+	FloatModel * m_subPhaseRand[64];
+	FloatModel * m_subSampLen[64];
+	FloatModel * m_subTempo[64];
+	FloatModel * m_subRateLimit[64];
+	FloatModel * m_subUnisonNum[64];
+	FloatModel * m_subUnisonDetune[64];
+	BoolModel * m_subInterpolate[64];
+
+	BoolModel * m_sampleEnabled[8];
+	BoolModel * m_sampleGraphEnabled[8];
+	BoolModel * m_sampleMuted[8];
+	BoolModel * m_sampleKeytracking[8];
+	BoolModel * m_sampleLoop[8];
+	FloatModel * m_sampleVolume[8];
+	FloatModel * m_samplePanning[8];
+	FloatModel * m_sampleDetune[8];
+	FloatModel * m_samplePhase[8];
+	FloatModel * m_samplePhaseRand[8];
+	FloatModel * m_sampleStart[8];
+	FloatModel * m_sampleEnd[8];
+
+	ComboBoxModel * m_modIn[64];
+	IntModel * m_modInNum[64];
+	FloatModel * m_modInAmnt[64];
+	FloatModel * m_modInCurve[64];
+	ComboBoxModel * m_modIn2[64];
+	IntModel * m_modInNum2[64];
+	FloatModel * m_modInAmnt2[64];
+	FloatModel * m_modInCurve2[64];
+	ComboBoxModel * m_modOutSec[64];
+	ComboBoxModel * m_modOutSig[64];
+	IntModel * m_modOutSecNum[64];
+	BoolModel * m_modEnabled[64];
+	ComboBoxModel * m_modCombineType[64];
+	BoolModel * m_modType[64];
+	BoolModel * m_modType2[64];
+
+	FloatModel * m_filtCutoff[8];
+	FloatModel * m_filtReso[8];
+	FloatModel * m_filtGain[8];
+	ComboBoxModel * m_filtType[8];
+	ComboBoxModel * m_filtSlope[8];
+	FloatModel * m_filtInVol[8];
+	FloatModel * m_filtOutVol[8];
+	FloatModel * m_filtWetDry[8];
+	FloatModel * m_filtBal[8];
+	FloatModel * m_filtSatu[8];
+	FloatModel * m_filtFeedback[8];
+	FloatModel * m_filtDetune[8];
+	BoolModel * m_filtEnabled[8];
+	BoolModel * m_filtMuted[8];
+	BoolModel * m_filtKeytracking[8];
+
+	FloatModel * m_macro[18];
+	QString m_macroTooltips[18] = {};
+	int m_macroColors[18][3] = {{0}};
+
+	FloatModel m_visvol;
+
+	FloatModel m_loadAlg;
+	FloatModel m_loadChnl;
+
+	IntModel m_mainNum;
+	IntModel m_subNum;
+	IntModel m_sampNum;
+	ComboBoxModel m_oversample;
+	ComboBoxModel m_oversampleMode;
+	ComboBoxModel m_loadMode;
+
+	graphModel m_graph;
+	
+	BoolModel m_visualize;
+
+	float m_storedwaveforms[8][STOREDMAINARRAYLEN] = {{0}};
+	float m_waveforms[8][MAINARRAYLEN] = {{0}};
+	bool m_mainFilled[8] = {false};
+	float m_storedsubs[64][STOREDSUBWAVELEN] = {{0}};
+	float m_subs[64][SUBWAVELEN] = {{0}};
```

Comment:
These large arrays make compilers use large memory, especially MSVC. I think we can fix AppVeyor CI by using something like `std::unique_ptr`, but I'm not sure if we really need it.